test/test.js

```diff
@@ -638,3 +638,47 @@ describe('httpProxyOptions.getProxyForUrl', function() {
       .expect(200, 'Response from https://example.com', done);
   });
 });
+
+describe('Custom helpFile', function() {
```

Comment:
As previously said, name this test after the property name for consistency:

```javascript
describe('helpFile', function() {
  ....
```

And you don't need two separate describe blocks, the following works equally well:

```javascript
describe('helpFile', function() {
  afterEach(stopServer);

  it('GET / with custom text helpFile', function() {
    cors_anywhere = ...
    request(...)...
  });

  ...
});
```